src/app/pages/users/tags/tags-list-table-data-source.ts

```diff
@@ -74,6 +74,11 @@ export class TagsListTableDataSource extends TableDataSource<Tag> {
       }
       this.loadUserFilterLabel(userID);
     }
+    const tagID = this.windowService.getSearch('tagID');
```

Comment:
tagID is in query param: first letter in uppercase -> TagID
I already renamed them also in your previous PR.
Don't hesitate to review my commits in your former PR.